src/bindings/lua/flux-lua.c

```diff
@@ -266,8 +265,11 @@ static int l_flux_index (lua_State *L)
     return 1;
 }
 
-static int send_json_request (flux_t *h, uint32_t nodeid, uint32_t matchtag,
-                              const char *topic, const char *json_str)
+static int send_json_request (flux_t *h,
```

Comment:
I'm kind of partial to how `black` puts even the first argument on its own line.
If others agree, `AlignAfterOpenBracket: AlwaysBreak` is the thing to add to `.clang-format`.

Reply:
I might not be remembering correctly, but I think we agreed the first argument on the same line as the function was more readable. (especially in C where the first arg is typically the "object" on which the "method" is being called)

Reply:
Yeah, I'm leaning towards @grondo's POV on that one.

Reply:
> (especially in C where the first arg is typically the "object" on which the "method" is being called)

That's a really great point!